css/app.css

```diff
@@ -0,0 +1,51 @@
+body{
+    background-color: slategray;
+}
+
+h1{
+    text-align: center;
+    text-decoration: underline;
+}
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.todo-list-container{
+    width: 400px;
+    padding: 30px;
+    background-color: whitesmoke;
```

Comment:
I love the CSS names. `Whitesmoke` would be a badass project name. `Project Whitesmoke: Where there's smoke, there's fire.`

Or, `Project Whitesmoke: A new Pope has been chosen`